changelog.md

```diff
@@ -480,6 +490,13 @@
 ### <a name="1.0.0"></a> 1.0.0
 - GA SDK.
 
+## Known regressions
```

Comment:
How about rename it to "Known Issues"? 
Regression reflect more alarming.

Reply:
Done